### ISSUE TITLE
`text-indent` not applied to Capsize elements

### ISSUE THREAD

**Issue by adamduncan** [seek-oss/capsize#76]
This probably falls into the same bucket as the "overflow/truncation" gotchas detailed in the FAQs. I was aware of the some of these layout-related constraints, but this caught me off guard today. Feels like something text-related that should've been fair game.

<img width="1550" alt="Screenshot 2022-02-22 at 11 56 41" src="https://user-images.githubusercontent.com/2671053/155127700-028e2c02-5ca3-400d-8de8-3f1eae38f1d6.png">

Like with truncation, this can be worked around by ensuring that the Capsize is applied to parent element and `text-indent` applied to its own separate child element.

Could be helpful to document?

**Comment by michaeltaranto** [comment]
Yeah any of these category of rules would have to be nested internally it seems. Will certainly consider how to incorporate this kind of advice in future documentation uplifts.

Thanks for sharing.
